src/js/components/day.jsx

```diff
@@ -52,11 +59,15 @@ export default class Day extends React.Component {
           description += ' - ';
         }
       }
+      let timeslotDate = momentTime.clone();
+      let status = moment(initialDate).isBefore(timeslotDate.add(slot[0], timeslotFormat)) ? DEFAULT : DISABLED;
```

Comment:
this should be a constant. Also, remember to send initialDate as moment object.

Finally, try to make code more readable:
```js
const timeslotDate = momentTime.add(slot[0], timeslotFormat);
let status = DEFAULT;
if (timeslotDate.isBefore(initialDate)) {
    status = DISABLED;
}
```